beacon_node/network/src/sync/network_context.rs

```diff
@@ -372,25 +358,100 @@ impl<T: BeaconChainTypes> SyncNetworkContext<T> {
         }
     }
 
+    fn active_request_count_by_peer(&self) -> HashMap<PeerId, usize> {
+        let Self {
```

Comment:
Might be worth mentioning why we use explicit field ignoring here, instead of { .. } - to prevent forgetting to add future request types.